lib/evss/disability_compensation_form/form4142_processor.rb

```diff
@@ -46,11 +46,12 @@ def initialize(submission, jid)
       # @return [Pathname] the stamped PDF path
       #
       def generate_stamp_pdf
+        submission_date = @submission.created_at.in_time_zone('Central Time (US & Canada)')
         pdf = PdfFill::Filler.fill_ancillary_form(
           form4142, @submission.submitted_claim_id, FORM_ID
         )
         stamped_path = CentralMail::DatestampPdf.new(pdf).run(text: 'VA.gov', x: 5, y: 5,
-                                                              timestamp: received_date)
```

Comment:
Sorry, I should have been more explicit. The utilized date would have to be before it was transformed by `strftime`.

Reply:
No you're good--I wasn't thinking. Latest commit is better :)